LambdaEngine/Source/ECS/ComponentManager.cpp

```diff
@@ -0,0 +1,20 @@
+#include "ECS/ComponentManager.h"
+
+using namespace LambdaEngine;
+
+LambdaEngine::ComponentManager::ComponentManager()
```

Comment:
Declare the constructor as default instead (in the header file, do: `ComponentManager() = default;`)